modules/common/tls/tls.go

```diff
@@ -67,6 +67,13 @@ const (
 
 	// TLSHashName - Name of the hash of hashes of all cert resources used to identify a change
 	TLSHashName = "certs"
+
+	// AdditionalSubjectNamesKey - Comma separated list of additionalSubjectNames
+	// that should be passed to the CertificateRequest
+	AdditionalSubjectNamesKey = "additionalSubjectNames"
```

Comment:
is this consistent with the common approach?

Reply:
yes lgtm